assets/js/components/Spinner.js

```diff
@@ -20,10 +20,10 @@
  * A single module. Keeps track of its own active state and settings.
  */
 
-function Spinner( { isSaving } ) {
+function Spinner( { isSaving, className } ) {
 	return (
 		<span
-			className="spinner"
+			className={ [ 'spinner', className ].join( ' ' ) }
```

Comment:
Please use the `classnames` package for dynamic class names. We already use it in many places and is our go-to utility for this kind of thing 👍

Reply:
Ah, perfect. Thank you